### PR TITLE
Feature:rating form & thank you page

### PR DESCRIPTION
**Pull request by samanehsan**
Add exp-ratings and exp-thank-you components to exp-addons

### CODE REVIEW

**Comment by samchrisinger** [comment]
@samanehsan can you please resolve merge conflicts?

**Comment by samanehsan** [comment]
Done making those updates @samchrisinger! ☔

File: exp-player/addon/components/exp-rating-form/component.js [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+            type: 'object',
+            properties: {
+                // define parameters here
+            }
+        },
+        data: {
+            type: 'object',
+            properties: {
+                questions: {
+                    default: questions
```
Comment (samchrisinger) [inline exp-player/addon/components/exp-rating-form/component.js]: Having this refer to a `questions` defined outside of the class instance feels risky to me. 

`data` defines what data is frame will export to be saved. Maybe it makes more sense to add a computed to fetch the actual values of the questions (minus some of the extra boilerplate?)?

**Comment by samanehsan** [comment]
Done making that change @samchrisinger! ☀️

**Pull request merged**